libs/db/go_level_db_test.go

```diff
@@ -6,9 +6,30 @@ import (
 	"fmt"
 	"testing"
 
+	"github.com/syndtr/goleveldb/leveldb/opt"
+
+	"github.com/stretchr/testify/assert"
 	cmn "github.com/tendermint/tendermint/libs/common"
 )
 
+func TestNewGoLevelDB(t *testing.T) {
+	name := fmt.Sprintf("test_%x", cmn.RandStr(12))
+	// Test write locks
+	db, err := NewGoLevelDB(name, "")
+	assert.Nil(t, err)
```

Comment:
I'd prefer `require` here which terminates the test (i.e. no need to continue it if we failed to open db)

Reply:
ACKd and incorporated.